src/disclosures/css/disclosures.less

```diff
@@ -1470,11 +1497,19 @@
     &__explore-schools {
       background-image: url('../img/explore-schools_300.png');
     }
+
+    &__work-while-studying {
+      background-image: url('../img/work-while-studying_300.png');
+    }
   }
 
   &_heading {
-    padding-top: unit(195px / @font-size, em);
+    padding-top: unit(225px / @font-size, em);
```

Comment:
Can you include a comment about where this number comes from for developers new to the codebase?